### DISCUSSION TITLE
Request: ExtPay trust badge

### DISCUSSION THREAD

**Discussion by ba32107** [Glench/ExtPay#218] [Ideas]
Hi,

I'd like to put a trust badge in my extension popup. Something that shows the user that I'm not handling any payments, not storing and credit card info, and they can trust the payment provider.

Basically I'm looking for this, but for ExtPay:
https://support.paymentsplugin.com/article/46-using-stripe-trust-badges

Since ExtPay is using Stripe under the hood, I _think_ I can get away with adding the Stripe trust badge. But it would be nice to have a dedicated, "official" trust badge for ExtPay that we can use. It could say something like "Powered by ExtensionPay & Stripe".

Related to this, it would be nice to have a section (or dedicated page) on https://extensionpay.com that explains to the user what this site is. Something with the title of:
"I use XYZ extension, I wanted to pay and I was redirected to this. What is this ExtensionPay thing?"

Then, the trust badge could link to this page.

Thanks!

**Comment by burkybang** [comment]
Technically you could place the badge there yourself. Your extension already has permission to execute a content script on ExtensionPay.com if you set it up correctly. Just use some JavaScript to place it on the page where you want it.

**Reply by ba32107** [reply]
I think you misunderstood me. I don't want to put anything on https://extensionpay.com/.

What I would like is a badge (an image), provided by ExtPay developers. If this was available, I would put this image into the popup of my extension.

The point of the badge is that I defer all responsibility of handling payments to ExtensionPay & Stripe, and I want my users to know this as well.

**Reply by burkybang** [reply]
![file-z9ooXSBSSC](https://github.com/Glench/ExtPay/assets/927886/f010f9ba-fff7-4b17-b0da-9b7849ec33c0)

**Reply by ba32107** [reply]
This works as a proof of concept, yes. But it doesn't work as a proper solution for multiple reasons:
* The badge should be produced & marketed by the ExtensionPay team. It only works if they stand behind it.
* This also modifies the original Stripe badge which defeats the purpose of it.

So I think this needs an original ExtensionPay image, then developers can put both that and the original Stripe one into their product.

**Reply by burkybang** [reply]
I don't think the average user is going to know or care if the image is marketed.
